Analysis/Tasks/PWGHF/HFJpsiToEECandidateSelector.cxx

```diff
@@ -222,5 +209,5 @@ struct HFJpsiToEECandidateSelector {
 WorkflowSpec defineDataProcessing(ConfigContext const&)
 {
   return WorkflowSpec{
-    adaptAnalysisTask<HFJpsiToEECandidateSelector>("hf-jpsi-candidate-selector")};
+    adaptAnalysisTask<HFJpsiToEECandidateSelector>("hf-jpsi-toee-candidate-selector")};
```

Comment:
this is ok with me if its ok with @vkucera too. we should keep these names consistent as well

Reply:
Vit have suggested it here https://github.com/AliceO2Group/AliceO2/pull/5085#discussion_r560856905

Reply:
Eventually, we should rename the classes, files and executables to follow the same naming scheme, so this will become something like `hf-candidate-selector-jpsi-toee`.
But let's keep that for a separate renaming PR.